tests/test_either.py

```diff
@@ -62,3 +62,9 @@ def test_either_flat_map_requires_callable(self):
     def test_either_repr(self):
         assert str(Right(random_int())).startswith("Right")
         assert str(Left(random_int())).startswith("Left")
+
+    def test_left_equality(self):
```

Comment:
also, might want to add some tests for inequality.  This is only testing what we call the "happy path".  Would probably want a test checking that equality can also be `False`.